Geometry/MTDGeometryBuilder/interface/RectangularMTDTopology.h

```diff
@@ -43,13 +43,14 @@ class RectangularMTDTopology final : public PixelTopology {
                          int ncols,
                          float pitchx,
                          float pitchy,
-                         bool upgradeGeometry,
-                         int ROWS_PER_ROC,       // Num of Rows per ROC
-                         int COLS_PER_ROC,       // Num of Cols per ROC
-                         int BIG_PIX_PER_ROC_X,  // in x direction, rows. BIG_PIX_PER_ROC_X = 0 for SLHC
-                         int BIG_PIX_PER_ROC_Y,  // in y direction, cols. BIG_PIX_PER_ROC_Y = 0 for SLHC
+                         int ROWS_PER_ROC,  // Num of Rows per ROC
+                         int COLS_PER_ROC,  // Num of Cols per ROC
                          int ROCS_X,
-                         int ROCS_Y)
+                         int ROCS_Y,
+                         float GAPxInterpad,
+                         float GAPxBorder,
+                         float GAPyInterpad,
+                         float GAPyBorder)
```

Comment:
Please add a comment here that the border and interpad values have to be in cm in the constructor.